BiasAndFairnessServers/src/crud/bias_and_fairness.py

```
@@ -85,3 +85,124 @@ async def delete_metrics_by_id(id: int, db: AsyncSession, tenant: str):
+    config_data: dict,
+    tenant: str,
+    db: AsyncSession
+):
+    """Insert a new bias and fairness evaluation."""
+    result = await db.execute(
+        text(f'''
+            INSERT INTO "{tenant}".bias_fairness_evaluations 
+            (eval_id, model_name, dataset_name, model_task, label_behavior, config_data) 
+            VALUES (:eval_id, :model_name, :dataset_name, :model_task, :label_behavior, :config_data) 
+            RETURNING id
+        '''),
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)